WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -188,6 +189,31 @@ public object GetStatus()
 		};
 	}
 
+	[JsonRpcMethod("getcoinjoinstatus")]
+	public string GetCoinjoinStatus()
+	{
+		var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+
+		AssertWalletIsLoaded();
+		var coinJoinManager = Global.HostedServices.Get<CoinJoinManager>();
+		try
+		{
+			var walletCoinjoinClientState = coinJoinManager.GetCoinjoinClientState(activeWallet.WalletName);
+			return walletCoinjoinClientState switch
+			{
+				CoinJoinClientState.Idle => "Idle",
+				CoinJoinClientState.InProgress => "In progress",
+				CoinJoinClientState.InSchedule => "In schedule",
+				CoinJoinClientState.InCriticalPhase => "In critical phase",
+				_ => throw new Exception($"The state {walletCoinjoinClientState.FriendlyName()} is unknown.")
```

Comment:
Here I didn't know whether to return `walletCoinjoinClientState.FriendlyName()` or to throw, but I think it makes more sense to `throw` to not hide if add a new `CoinJoinClientState` but forgot to add it here.